### PR TITLE
Further manual edits

### PR DESCRIPTION
**Pull request by prckent**

### CODE REVIEW

File: manual/backflow_implementation.tex [hunk 1]
```diff
@@ -30,31 +30,31 @@ \subsection{Value}
   ,WorkSpace.data(),Pivot.data(),PhaseValue);
 \end{lstlisting}
 
-QMCPACK represents the complex value of the wavefunction in polar coordinates $S=e^Ue^{i\theta}$. Specifically, \verb|LogValue| $U$ and \verb|PhaseValue| $\theta$ are handled separately. In the following, I will consider derivatives of the log value only.
+QMCPACK represents the complex value of the wavefunction in polar coordinates $S=e^Ue^{i\theta}$. Specifically, \verb|LogValue| $U$ and \verb|PhaseValue| $\theta$ are handled separately. In the following, we will consider derivatives of the log value only.
 
 \subsection{Gradient}
-To evaluate particle gradient of the log value of the Slater-Backflow wavefunction, we can use the $\log\det$ identity in eq.~\ref{eq:logdet}. This identity maps the derivative of $\log\det M$ with respect to a real variable $p$ to a trace over $M^{-1}dM$
+To evaluate particle gradient of the log value of the Slater-backflow wavefunction, we can use the $\log\det$ identity in Equation~\ref{eq:logdet}. This identity maps the derivative of $\log\det M$ with respect to a real variable $p$ to a trace over $M^{-1}dM$:
```
Comment (ye-luo) [inline manual/backflow_implementation.tex]: May be it is more than necessary. I prefer brackets around equation numbers like (3).

Comment (prckent) [reply manual/backflow_implementation.tex]: Equations references are currently without () throughout the manual.

**Comment by prckent** [comment]
Fixed a few inconsistencies and typos.

**Comment by prckent** [comment]
I have just received the appendices; will add here momentarily.

**Pull request merged**